djstripe/managers.py

```diff
@@ -7,6 +7,18 @@
 from django.db import models
 
 
+class StripeObjectManager(models.Manager):
+    def exists_by_json(self, data):
+        """
+        Search for a matching stripe object based on a Stripe object
+        received from Stripe in JSON format
+        :param data: Stripe event object parsed from a JSON string into an object
+        :type data: dict
+        :rtype: bool
+        """
```

Comment:
Will do on the first point.

On the second, I'm viewing the collection of classes in stripe_objects as just being our local representation of Stripe's data model. That's the reason I refer to it as a 'stripe' object.